<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>811</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Video processing</string>
     </attribute>
     <widget class="QPushButton" name="btnDefaultVideo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Default Video</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="videoDisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>781</width>
        <height>501</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHog">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>560</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>run People Detector</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_blockProcedure">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>560</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>run Background Segmentator</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMainFlow">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>run Main Flow</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image processing </string>
     </attribute>
     <widget class="QGraphicsView" name="imgDisplay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>501</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Bilateral Filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_def_img">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>600</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Default Image</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionOtro_Abrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir_Imagen">
   <property name="text">
    <string>Abrir Imagen</string>
   </property>
  </action>
  <action name="actionOtro_Abrir">
   <property name="text">
    <string>Abrir Imagen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
